sa/model.go

```diff
@@ -281,12 +281,16 @@ type regModel struct {
 	Agreement string `db:"agreement"`
 	// InitialIP is stored as sixteen binary bytes, regardless of whether it
 	// represents a v4 or v6 IP address.
+	//
+	// Deprecated: This field is no longer used and will be removed in a future.
```

Comment:
Don't just delete the deprecation notice. The comment needs to be updated to explain why this field still exists even though we're not using it anywhere else.

Reply:
I'm a little confused by this comment. I've added a deprecation notice, not deleted one.

Reply:
Wow, I am clearly awake.

I think I read this with the _expectation_ that this PR was going to be replacing "mere" deprecation with actual deletion. The first version of this comment I almost left was "you forgot to delete the field!" before I remembered that we actually still needed it so that we could insert dummy addresses.

So this comment is fine, but I think could be further improved. That field _is_ still used inside registrationPBToModel, and therefore inside NewRegistration, and it will continue to be used there until we move this table to vitess.